### PR TITLE
[TE] Self-Serve Edit MVP2

### PR DESCRIPTION
**Pull request by newsummit**
This introduces logic for properly editing alert and config group entities in ThirdEye Self-Serve Alert Edit UI.

Also: bug fix for Create Alert (missing crontab property)

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/manage/alerts/edit/route.js [hunk 1]
```diff
@@ -59,26 +63,30 @@ export default Ember.Route.extend({
...
+      })
+      .then((groupByAlertId) => {
+        const originalConfigGroup = groupByAlertId ? groupByAlertId.pop() : null;
+        const selectedAppName = originalConfigGroup ? originalConfigGroup.application : null;
+        Object.assign(model, { originalConfigGroup, selectedAppName });
+        return fetch('/thirdeye/entity/APPLICATION').then(checkStatus);
+      })
+      .then((allApps) => {
+        Object.assign(model, { allApps });
       });
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/manage/alerts/edit/route.js]: nit: do we need to add a `.catch` , can any of the calls above fail?

File: thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/template.hbs [hunk 2]
```diff
@@ -115,9 +115,9 @@
               </span>
             </div>
             {{!-- TODO: add edit button once MVP is finalized --}}
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/template.hbs]: let's delete this

File: thirdeye/thirdeye-frontend/app/pods/manage/alerts/edit/route.js (hunk 1 above)
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/pods/manage/alerts/edit/route.js]: Yes - we should add one, for sure.

File: thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/template.hbs (hunk 2 above)
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/template.hbs]: oops

**Pull request merged**